lib/sections/driver.js

```diff
@@ -1,60 +1,88 @@
 var utils = require(__dirname + '/utils.js');
 
-var driverJob = function(){
+var busJob = function(){
   return {
-    plugins: [],
-    pluginOptions: [],
     perform: function(args, callback){
       var self = this;
 
-      var started = 0;
-      var matches = [];
-      self.subscriptions(function(err, subscriptions, count){
-        if(count == 0){
-          if(started === 0 && typeof callback === 'function'){ callback(null, matches); }
-        }else{    
-          for(var app in subscriptions){
-            for(var i in subscriptions[app]){
-              started++;
-              var subscription = subscriptions[app][i];
-              var matched = subscriptionMatch(args, subscription);
-              if(matched === true){
-                matches.push(subscription['queue_name']);
-                var payload = driverMetadata(args, subscription.queue_name, app, i, subscription.class, args.bus_event_type);
-                self.queueObject.enqueue(subscription.queue_name, subscription.class, payload, function(err, toRun){
-                  started--;
-                  if(started === 0 && typeof callback === 'function'){ callback(null, matches); }
-                });
-              }else{
-                process.nextTick(function(){
-                  started--;
-                  if(started === 0 && typeof callback === 'function'){ callback(null, matches); }
-                });
-              }
-            }
+      if(args.bus_class_proxy === '::QueueBus::Driver'){   
```

Comment:
for safety, it would be best to also handle these without the colons. e.g. `QueueBus::Driver`